app/helpers/application_helper.rb

```diff
@@ -35,6 +35,6 @@ def active_scope_filter(title, options = {})
 
   def search_tab_options(options, tab)
     options.delete(:page)
-    options.merge(tab: tab) unless tab == 'All'
+    tab.eql?('All') ? options : options.merge(tab: tab)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.